lib/solidus_shipwire/engine.rb

```diff
@@ -10,6 +11,14 @@ class Engine < Rails::Engine
       g.test_framework :rspec
     end
 
+    initializer 'solidus_shipwire.environment', before: 'spree.environment' do
+      Spree::SolidusShipwireConfig = Spree::SolidusShipwireConfiguration.new
```

Comment:
why not use `Spree::ShipwireConfig`? If one day`Spree` namespace will be converted into `Solidus` we'll have `Solidus::SolidusShipwireConfig`